apis/placement/v1/clusterresourceplacement_types.go

```diff
@@ -500,7 +500,7 @@ type RollingUpdateConfig struct {
 	// Absolute number is calculated from percentage by rounding up.
 	// We consider a resource unavailable when we either remove it from a cluster or in-place
 	// upgrade the resources content on the same cluster.
-	// This can not be 0 if MaxSurge is 0.
+	// The minimum of MaxUnavailable is 1 to avoid rolling out stuck during in-place resource update.
```

Comment:
// The desired number equals to the `NumberOfClusters` field when the placement type is `PickN`.
	// The desired number equals to the number of clusters scheduler selected when the placement type is `PickAll`.
	
What is the desired number? Why does it equal to max possible number?

Reply:
it is used to calculate the real number when the maxUnavailable is a percent like 25%